src/main/java/nebula/test/dsl/TestProjectRunner.java

```diff
@@ -0,0 +1,58 @@
+package nebula.test.dsl;
+
+import org.gradle.testkit.runner.BuildResult;
+import org.gradle.testkit.runner.GradleRunner;
+import org.jspecify.annotations.NullMarked;
+
+import java.io.File;
+import java.util.List;
+
+@NullMarked
+public class TestProjectRunner {
```

Comment:
Wonder if it make sense to port some feature such as `isJwdpLoaded` in https://github.com/nebula-plugins/nebula-test/blob/main/src/main/groovy/nebula/test/IntegrationTestKitBase.groovy#L107

Reply:
for sure. I can start working on a list of features in the existing fixtures to track porting/sharing them